Remove beta

groovecoder opened the pull request.

pdehaan reviewed (commented):
Sorry, I don't think I can inline comment on image/binary diffs, so accept this odd mash of out of context nonsense:

extension/icons/make-new-alias.png resized from 150&times;150px to 262&times;287px. The extra 5.6KB filesize isn't entirely unexpected, but still a bit strange that the image dimensions changed as much as they did.

Minor nit: static/images/email-images/relay-beta-logo-emails.png still has "Beta" in the filename. 

static/images/placeholder-logo.svg has a weird right padding now (where the old "BETA" ribbon would have been). Not sure if this is worth resizing, or wait and see when it's deployed into dev servers if there is any noticeable UI layout/padding issues.

groovecoder commented on .env-dist:
```diff
@@ -20,6 +20,7 @@ SOCKETLABS_API_KEY="dummy-value"
 SOCKETLABS_VALIDATION_KEY="dummy-value"
 RELAY_FROM_ADDRESS="relay@127.0.0.1:8000"
 SITE_ORIGIN="http://127.0.0.1:8000"
+MAX_NUM_FREE_ALIASES=10
```
oops. that's actually just the `dist` file - limit will still be 5 in prod.

pdehaan replied:
💔

groovecoder commented on extension/js/fill_relay_address.js:
```diff
@@ -23,14 +23,14 @@ async function showModal(modalType) {
   const manageAliasesLink = document.createElement("a");
   manageAliasesLink.textContent = "Manage All Aliases";
   manageAliasesLink.classList = ["fx-relay-new-tab fx-relay-modal-manage-aliases"];
-  manageAliasesLink.href = `${relaySiteOrigin}?utm_source=fx-relay-addon&utm_medium=context-menu-modal&utm_campaign=beta&utm_content=manage-relay-addresses`;
```
Nah, we don't have any metrics reports depending on campaign value.

groovecoder commented on extension/js/background.js:
```diff
@@ -157,7 +157,7 @@ browser.runtime.onMessage.addListener(async (m) => {
     case "openRelayHomepage":
       browser.tabs.create({
-        url: `${RELAY_SITE_ORIGIN}?utm_source=fx-relay-addon&utm_medium=input-menu&utm_campaign=beta&utm_content=go-to-fx-relay`,
+        url: `${RELAY_SITE_ORIGIN}?utm_source=fx-relay-addon&utm_medium=input-menu&utm_content=go-to-fx-relay`,
```
Do you mind filing a separate issue to clean up this inconsistency?

groovecoder commented:
I think I addressed all comments except:

> static/images/placeholder-logo.svg has a weird right padding now (where the old "BETA" ribbon would have been). Not sure if this is worth resizing, or wait and see when it's deployed into dev servers if there is any noticeable UI layout/padding issues.

I was hand-editing the SVGs via dev tools inspector. @maxxcrawford - any chance you could edit this file to remove the BETA ribbon while keeping the proper alignment?

say-yawn reviewed (approved):
CODE LGTM

groovecoder commented:
Updated with new and better images from @maxxcrawford . @pdehaan - do you want one last review of this before I merge?

pdehaan commented:
Also ran <kbd>git grep -irnI "beta"</kbd> which implies there were no [case-insensitive] lingering references to "beta" text in the repo.
